src/sentry/issues/endpoints/group_details.py

```diff
@@ -24,10 +24,10 @@
 from sentry.api.serializers.models.group_stream import get_actions, get_available_issue_plugins
 from sentry.api.serializers.models.plugin import PluginSerializer
 from sentry.api.serializers.models.team import TeamSerializer
-from sentry.incidents.utils.metric_issue_poc import OpenPeriod
 from sentry.integrations.api.serializers.models.external_issue import ExternalIssueSerializer
 from sentry.integrations.models.external_issue import ExternalIssue
 from sentry.issues.constants import get_issue_tsdb_group_model
+from sentry.issues.endpoints.group_open_periods import get_open_periods_for_group
```

Comment:
let's make a helper/utils file in issues/endpoints/ so we don't have to import another endpoint file from here

Reply:
actually, since this function is doing something for a group, it can live on the `Group` model.